README.md

```diff
@@ -1,358 +1,48 @@
-3. Run the following ballerina command module-ballerinax-redis directory:
-
-    3.1 If running on a Unix OS:
-         
-        bal build redis
-    3.2 The unit tests will not run in Windows OS. Therefore, if running on a Windows OS skip the tests using following command:
-         
-        bal build -c --skip-tests redis
-         
-## Compatibility
-
-|                             |           Version           |
-|:---------------------------:|:---------------------------:|
-|      Ballerina Language     |      Swan Lake Alpha 5      |
-
-```ballerina
-import ballerinax/redis;
-import ballerina/io;
-
-redis:ClientEndpointConfiguration redisConfig = {
-        host: "localhost:6379",
-        password: "",
-        options: { connectionPooling: true, isClusterConnection: false, ssl: false,
-            startTls: false, verifyPeer: false, connectionTimeout: 500 }
-};
-
-public function main() returns error? {
-    redis:Client conn =  check new (redisConfig);
-
-    io:println("Pinging Redis Server...");
-    //Ping Server
-    var result = conn->ping();
-    if (result is string) {
-        io:println(result);
-    } else {
-        io:println(result.message());
-    }
-
-    io:println("===Executing sample string oprerations===");
-    //Sample String Operations
-    io:println("Setting value of the key \"Project\" as \"Ballerina\"");
-    var stringSetresult = conn->set("Project", "Ballerina");
-
-    if (stringSetresult is string) {
-        io:println("Reply from the server: " + stringSetresult);
-    } else {
-        io:println("Error occurred while calling `set`");
-    }
-
-    io:println("\nQuerying the server for the value of the key \"Project\"");
-    var value = conn->get("Project");
-
-    if (value is string) {
-        io:println("Reply from the server: " + value);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `get`");
-    }
-
-    io:println("\n===Executing sample list oprerations===");
-    //Sample List Operations
-    io:println("Pushing 2 elements to NumberList");
-    var listPushresult = conn->lPush("NumberList", ["One", "Two"]);
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-    if (listPushresult is int) {
-        io:println(io:sprintf("Number of elements pushed: %s", listPushresult));
-    } else {
-        io:println("Error occurred while calling `lPush`");
-    }
+[Redis](https://redis.io/) is an open source, in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets.
 
-    io:println("Poping an element from NumberList");
-    var lPopResult = conn->lPop("NumberList");
+This connector provides operations for connecting and interacting with Redis server over the network. 
+For more information about configuration and operations, go to the module. 
```

Comment:
Suggestion:
For more information on connector configuration and operations, see [`ballerinax/redis`](redis/Module.md).